src/main/java/seedu/duke/chatbot/Ui.java

```diff
@@ -0,0 +1,146 @@
+package seedu.duke.chatbot;
+
+import seedu.duke.task.Task;
+import seedu.duke.task.TaskList;
+
+import java.io.File;
+import java.util.Scanner;
+
+/**
+ * Handles all interactions with the user.
+ * Takes in user input using a Scanner object.
+ * Specifies the outputs from Duke in different scenarios.
+ */
+public class Ui {
+    private final Scanner sc;
+
+    /**
+     * Creates a Ui object that can take in commands from the user.
+     */
+    Ui() {
```

Comment:
Maybe add a public prefix to the methods? I noticed that some of your Ui methods don't have that while the rest of the methods have them.